### PR TITLE
Settings: Hindi Translation For Derpfest Additions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by idoybh — APPROVED** [review]

File: res/values-hi/aosip_strings.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+    <!-- Sound: Title for the option defining the phone ringtone for slot2 on Muilti SIM device. [CHAR LIMIT=50] -->
+    <string name="ringtone2_title">फोन रिंगटोन - सिम 2</string>
+
+    <!-- Charging vibration -->
+    <string name="charging_vibro_title">चार्जिंग कंपन</string>
+
+    <!-- Notification sound timeout -->
+    <string name="app_notification_sound_timeout_title">सूचना ध्वनियों के बीच न्यूनतम समय</string>
+    <string name="app_notification_sound_timeout_summary_template">ध्वनियों या कंपन को एक से अधिक बार हर <xliff:g id="duration">%1$s</xl:> में अनुमति दें</string>
```
Comment (idoybh) [inline res/values-hi/aosip_strings.xml]:
```suggestion
    <string name="app_notification_sound_timeout_summary_template">ध्वनियों या कंपन को एक से अधिक बार हर <xliff:g id="duration">%1$s</xliff:g> में अनुमति दें</string>
```
This was not done correctly

**Review by idoybh — CHANGES_REQUESTED** [review]

File: res/values-hi/aosip_strings.xml (hunk 1 above)
Comment (ghost) [reply res/values-hi/aosip_strings.xml]: Yupp, messed up there but the last three words should be there.

Comment (idoybh) [reply res/values-hi/aosip_strings.xml]: edited. like this?

Comment (ghost) [reply res/values-hi/aosip_strings.xml]: Yes, thats better now

**Review by idoybh — APPROVED** [review]

**Pull request merged**